### ISSUE TITLE
This software is not supported on your system

### ISSUE THREAD

**Issue by pigsyn** [pigsyn/MacProFirmwareToolUpdate#4]
running through the steps here and I get the message: 

"This software is not supported on your system."

I'm on an early '09 4,1. I ran the csrutil disable which told me it was successful. Sort of at a dead end as far as I can tell. Anyone else run into this?

_Originally posted by @SEIZMICdesign in https://github.com/pigsyn/MacProFirmwareToolUpdate/issues/3#issuecomment-582720243_

**Comment by pigsyn** [comment]
Hello SEIZMICdesign could you provide your macOS version as well as a screenshot of Hardware Overview from Apple System Informations ?
![Apple System Informations](https://help.apple.com/assets/5D96982D680CE2FB3A9F622E/5D96983F680CE2FB3A9F6235/fr_FR/c932e32eb5e894dc134de6d00e04e922.png)

**Comment by SEIZMICdesign** [comment]
Thanks for jumping on this, but I figured it out! My stumbling block was that it's not precisely clear that the Apple updater isn't supposed to *actually* work, you just need it open while you run the 4,1 to 5,1 app. Shortly after I posted that it just kinda came to me. Might want to make the instructions a little more clear in that regard for others wanting to do this update.

Now to drop in the 5690s! wish me luck!!!
